File: JavaGoogleChart/srctests/com/a2a/googlechart/ScatterPlotTests.cs
package com.a2a.googlechart;

namespace Tests
{
    class ScatterPlotTests
    {
        public static string scatterPlotTest()
        {
            int[] xData = { 10, 20, 30, 40, 50 };
            int[] yData = { 10, 20, 30, 40, 50 };

            List<int[]> dataset = new List<int[]>();
            dataset.Add(xData);
            dataset.Add(yData);

            ScatterPlot scatterPlot = new ScatterPlot(150, 150);
            scatterPlot.SetTitle("Scatter Plot");
            scatterPlot.SetData(dataset);

            scatterPlot.AddAxis(new ChartAxis(ChartAxisType.Left));
            scatterPlot.AddAxis(new ChartAxis(ChartAxisType.Bottom));

            return scatterPlot.GetUrl();
        }
    }
}

File: trunk/JavaGoogleChart/srctests/com/a2a/googlechart/ScatterPlotTests.java
package com.a2a.googlechart;

import com.a2a.googlechart.charts.ScatterPlot;
import com.a2a.googlechart.data.VectorInt;

public class ScatterPlotTests
{
    public static String scatterPlotTest()
    {
        int[] xData = { 10, 20, 30, 40, 50 };
        int[] yData = { 10, 20, 30, 40, 50 };

        VectorInt dataset = new VectorInt();
        dataset.add(xData);
        dataset.add(yData);

        ScatterPlot scatterPlot = new ScatterPlot(150, 150);
        scatterPlot.setTitle("Scatter Plot");
        scatterPlot.setData(dataset);

        scatterPlot.addAxis(new ChartAxis(ChartAxis.Left));
        scatterPlot.addAxis(new ChartAxis(ChartAxis.Bottom));

        return scatterPlot.getUrl();
    }
}
